k8s/cert-manager/kustomization.yaml

```diff
@@ -2,4 +2,29 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - issuers.yaml
-  - certificate.yaml
+patches:     
+- target:
+    group: cert-manager.io
+    version: v1
+    kind: ClusterIssuer
+    name: letsencrypt
+    namespace: istio-system
```

Comment:
Since it's a `ClusterIssuer`, I don't think we'll be needing this.

Reply:
Good catch! fixed and pushed